lib/ref_arch_setup/bolt_helper.rb

```diff
@@ -128,5 +185,23 @@ def self.upload_file(source, destination, nodes)
 
       return success
     end
+
+    # Install modules from the forge via Puppetfile
+    #
+    # @author Bill Claytor
+    #
+    # @return [true,false] Based on exit status of the bolt task
+    def self.install_forge_modules
+      command = "bolt puppetfile install --modulepath #{FORGE_MODULE_PATH}"
+      puts "Running: #{command}"
```

Comment:
feels like we are repeating a pattern here... we set the command, and then do the almost the exact same thing in 3 methods...

Reply:
SLV-240